playbooks/aws/openshift-cluster/vars.yml

```diff
@@ -37,3 +37,15 @@ deployment_vars:
     security_groups: [ 'public' ]
     vpc_subnet:
     assign_public_ip:
+  openshift-enterprise:
```

Comment:
I tried something like:
```
deployment_vars:
  rhel7_ent_base:
    …
  enterprise: "{{ rhel7_ent_base }}"
```
but this produces:
```
fatal: [localhost] => One or more undefined variables: 'rhel7_ent_base' is undefined
```

So, I tried:
```
deployment_vars:
  rhel7_ent_base:
    …
  enterprise: "{{ deployment_vars.rhel7_ent_base }}"
```
but this produces:
```
ERROR: recursive loop detected in template string
```

That’s why I had to define the “common settings” variable at the top level.
Would you know if I missed another Ansible trick?